server/api/products.js

```diff
@@ -0,0 +1,38 @@
+const router = require('express').Router()
+//Require product model
+const {Product, User, Order} = require('../db/models/')
+module.exports = router
+
+//FInd all products
+router.get('/', async (req, res, next) => {
+  try {
+    const products = await Product.findAll({
+      //Eager loading
+      include: [{model: User}, {model: Order}]
```

Comment:
If this route is just for display all products then I don't think we need to eager load User and Order here information here. I think that any user that accessed this route would get user and order info that they shouldn't have access to.